scene/resources/curve.cpp

```diff
@@ -479,6 +479,9 @@ void Curve::set_bake_resolution(int p_resolution) {
 }
 
 real_t Curve::sample_baked(real_t p_offset) const {
+	// Make sure that p_offset is finite.
+	ERR_FAIL_COND_V_MSG(!Math::is_finite(p_offset), 0, "Offset is non-finite");
+	
```

Comment:
There's trailing whitespace on these lines, needs to be removed

Reply:
Thanks, I didn't have clang-format set up locally before so I missed it, should be good now 🙂.